pkgs/development/tools/cppclean/default.nix

```diff
@@ -3,14 +3,14 @@
 with python3Packages;
 
 buildPythonApplication rec {
-  name = "cppclean-unstable-${version}";
-  version = "2018-05-12";
+  name = "cppclean-${version}";
+  version = "v0.13";
```

Comment:
Please move the v prefix to ``rev`` and also consider using ``pname``.

Reply:
Done (thanks, @jonringer::) )